### PR TITLE
Update kp.m3u

### PR DESCRIPTION
**Pull request by taodicakhia**
Update:
- elufatv's Twitch channel is changed into kctv_elufa (https://www.twitch.tv/kctv_elufa)
- The m3u8 stream link of tv.nknews.org (https://tv.nknews.org/tvhls/stream.m3u8) is not playable (requires referrer for link and the content of link)

### CODE REVIEW

File: channels/kp.m3u [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #EXTM3U
+#EXTINF:-1 tvg-id="KCTV.kp" tvg-country="KP" tvg-language="Korean" tvg-logo="https://i.imgur.com/rEn31ub.png" group-title="",Korean Central Television (KCTV) (810p)
+https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=htps://www.twitch.tv/kctv_elufa
```
Comment (freearhey) [inline channels/kp.m3u:3]:
```suggestion
https://query-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.twitch.tv/kctv_elufa
```

**Review by freearhey — CHANGES_REQUESTED** [review]

**Review by Sphinxroot — APPROVED** [review]

**Comment by Dum4G** [comment]
We can add a refferer if you know what it is. tv.nknews.org is the right one?

**Review by freearhey — APPROVED** [review]

**Comment by freearhey** [comment]
@taodicakhia Thank you for help!

**Comment by taodicakhia** [comment]
> We can add a refferer if you know what it is. tv.nknews.org is the right one?
Yes

**Pull request merged**